### ISSUE TITLE
language phrase search character problem

### ISSUE THREAD

**Issue by perayknox** [moxi9/phpfox#887]
hi

when I search a phrase in language section on admin section, I can't find that I searched phrase because there is a character problem.

http://mywebsite.com/index.php/admincp/language/phrase/?lang-id=tr

search with turkish character like ş, ü, ç, ğ, İ, ö, 

it can't find anything. but there is a phrase like this: şaka, çocuk, İkna, ağa. :)

**Comment by perayknox** [comment]
![2015-12-29_21-59-15](https://cloud.githubusercontent.com/assets/13890542/12041163/70ac00c6-ae77-11e5-9a07-74670af6d0a8.gif)

**Comment by Data66PF** [comment]
confirmed. I have the turkish pack installed and went to that pack and clicked Manage Phrases so that all I would be seeing is that pack's phrases. I then tried to search various things Bloglar works and anything without special characters. I then looked to find phrases in it (manually) with special characters and none of those work when I copy and paste them into the search box. Example: özellik returns nothing but it is in a phrase privacy.user_setting_can_set_allow_list_on_blogs
